Block/Bordered/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react'
+import compose from 'ramda/src/compose'
+import classNamesBind from 'classnames/bind'
+import childrenPropType from '../../propTypes/children'
+import defaultStyles from './styles.scss'
+import {
+  overridable,
+  themeable,
+  uniqueName
```

Comment:
No. Removed it from the implementation, so should not be part of the imports. Will fix.